presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py

```diff
@@ -19,24 +19,25 @@ def fig2img(fig):
 class ImagePiiVerifyEngine:
     """ImagePiiVerifyEngine class only supporting Pii verification currently."""
 
-    def __init__(self, analyzer_engine: Optional[ImageAnalyzerEngine] = None):
-        if not analyzer_engine:
-            analyzer_engine = ImageAnalyzerEngine()
-        self.analyzer_engine = analyzer_engine
+    def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = None):
+        if not image_analyzer_engine:
+            image_analyzer_engine = ImageAnalyzerEngine()
+        self.image_analyzer_engine = image_analyzer_engine
```

Comment:
```suggestion
    def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = ImageAnalyzerEngine()):
            self.image_analyzer_engine = image_analyzer_engine
```

Reply:
@SharonHart I wouldn't recommend this https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/#:~:text=In%20Python%2C%20when%20passing%20a,or%20even%20a%20class%20instance.

Reply:
Nice to know, fail enough :)